merge_data/tourism_FR.mapping.json

```
@@ -11,5 +11,42 @@
       "information": "office",
       "tourism": "information"
     }
+  },
+  {
+    "title": "Church",
+    "type": "https://www.datatourisme.gouv.fr/ontology/core#Church",
+    "items": 8101,
+    "classes": 310,
```

Increment each by 10.
```suggestion
    "classes": 320,
```